src/parser/parser_driver.cpp

```diff
@@ -441,7 +441,21 @@ void ParserDriver::defineGrammar()
 				return {};
 			},
 			"tags", "LCB", "metas", "strings", "condition", "RCB", [&](auto&& args) -> Value {
-				std::optional<TokenIt> mod = std::move(args[0].getOptionalTokenIt());
+				std::optional<TokenIt> mod_private = {};
+				std::optional<TokenIt> mod_global = {};
+				const std::vector<TokenIt> mods = std::move(args[0].getMultipleTokenIt());
+				for (const auto &token: mods) {
+					if (token->getType() == TokenType::GLOBAL) {
+						if (mod_global.has_value())
+							error_handle(token->getLocation(), "Duplicated global rule modifier");
+						mod_global = token;
+					}
+					else if (token->getType() == TokenType::PRIVATE) {
```

Comment:
If you don't mind, I would put the bracket on a new line.

Reply:
Sure! I tried to be consistent with the style, but muscle memory won in this case.